package.json

```diff
@@ -36,12 +36,12 @@
     "doc:gem:install": "cd docs/ && bundle install",
     "lint": "esw src/ -w --ext .ts --ext .tsx",
     "lint:once": "eslint src/ --ext .ts --ext .tsx",
-    "lint:fix": "eslint src/ --ext .ts --ext .tsx --fix",
+    "lint:fix": "eslint src/ --ext .ts --ext .tsx --cache --fix",
```

Comment:
What does the --cache option do?

Reply:
`--cache` only checks changed files, we don't need to check the whole project every time.